src/resources/authoring/gv2.json

```diff
@@ -370,14 +370,14 @@
       "instructions": "Create 5 different baby drakes!",
       "showUserDrake": true,
       "mother":{
-        "alleles": "w-W, M-m, fl-, hl-, T-T, h-h, C-C, A1-A1, B-B, D-D, rh-rh, Bog-Bog",
+        "alleles": "w-W, M-M, fl-, hl-, T-T, h-, C-C, A1-a, B-B, D-D, rh-rh, Bog-Bog",
```

Comment:
Lots of changes here to match updated specs, including a totally flattened level structure for now